Makefile

```diff
@@ -1,9 +1,19 @@
-make test_backend:
+test_backend:
 	docker-compose run --rm backend python3 manage.py test && pylint apps
 
-make test_frontend:
+test_frontend:
 	docker-compose run --rm frontend node_modules/.bin/ng test --single-run
 
-make test_all:
-	make test_backend
-	make test_frontend
+test_all: test_backend test_frontend
+
+populate_database:
+	docker-compose run --rm backend python3 manage.py populate_database
+
+# Beware - Following commands delete ALL docker images and containers!
+docker_remove_containers:
```

Comment:
if that's in `Makefile` I would suggest to do here
```
docker rm $$(docker ps -qa | grep volontulo_)
```
to only remove volontulo related images/containers.

Reply:
`docker ps -a | grep 'redis' | awk '{print $1}' | xargs docker rm -vf`

Reply:
Or even nicer  `docker container prune` and filter by image name (I don't know if that's possible though).

Reply:
As far as I could investigate it , "docker container prune" does not show any possibilities for filtering. However, your comment inspired me to search for 'native' docker filtering instead of unix pipes :)